### PR TITLE
chore: service accounts GA

### PR DESCRIPTION
**Pull request by gastonfournier**
## About the changes
This PR prepares the GA of service accounts: OpenAPI tags, documentation and flag removal

Relates to [roadmap](https://github.com/orgs/Unleash/projects/10) item: #2942

### CODE REVIEW

File: src/lib/__snapshots__/create-config.test.ts.snap [hunk 1]
```diff
@@ -104,7 +104,7 @@ exports[`should create default config 1`] = `
       "projectStatusApi": false,
       "proxyReturnAllToggles": false,
       "responseTimeWithAppName": false,
-      "serviceAccounts": false,
+      "serviceAccounts": true,
```
Comment (nunogois) [inline src/lib/__snapshots__/create-config.test.ts.snap:107]: Could be a mistake from having the ENV variable set locally. Try recreating the snap with it off?

Comment (gastonfournier) [reply src/lib/__snapshots__/create-config.test.ts.snap:107]: Not exactly sure why this happens but it's not because of a local variable... :thinking:

Comment (gastonfournier) [reply src/lib/__snapshots__/create-config.test.ts.snap]: I just need to remember pushing my local branch :D

**Review by ivarconr — APPROVED** [review]

**Comment by nunogois** [comment]
Other places where we may want to clean up the flag:

https://github.com/Unleash/unleash/blob/80ed90e7404e940f591500e1912cb375c519432e/frontend/src/component/admin/menu/AdminMenu.tsx#L38

https://github.com/Unleash/unleash/blob/80ed90e7404e940f591500e1912cb375c519432e/frontend/src/hooks/api/getters/useServiceAccountTokens/useServiceAccountTokens.ts#L18

https://github.com/Unleash/unleash/blob/80ed90e7404e940f591500e1912cb375c519432e/frontend/src/interfaces/uiConfig.ts#L46

File: src/lib/openapi/util/openapi-tags.ts [hunk 2]
```diff
@@ -18,6 +18,11 @@ const OPENAPI_TAGS = [
         description:
             'Create, update, and delete [Personal access tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys#personal-access-tokens).',
     },
+    {
+        name: 'Service Accounts',
+        description:
+            'Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts) enabling apps to access Unleash API programatically.',
```
Comment (nunogois) [inline src/lib/openapi/util/openapi-tags.ts:24]: Small typo in `programatically` - `programmatically`.

Small nitpick but the sentence doesn't read very well to me, maybe something like this instead?

```suggestion
            'Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts), which enable programmatic access to the Unleash API.',
```

File: src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap [hunk 3]
```diff
@@ -8423,6 +8423,10 @@ If the provided project does not exist, the list of events will be empty.",
       "description": "Create, update, delete, and manage [segments](https://docs.getunleash.io/reference/segments).",
       "name": "Segments",
     },
+    {
+      "description": "Endpoints for managing [Service Accounts](https://docs.getunleash.io/reference/service-accounts) enabling apps to access Unleash API programatically.",
+      "name": "Service Accounts",
+    },
```
Comment (nunogois) [inline src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap:8429]: If we're updating the tag description we should update the snap as well.

File: website/docs/how-to/how-to-create-service-accounts.mdx [hunk 4]
```diff
@@ -4,7 +4,7 @@ title: How to create service accounts
 
 :::info availability
 
-Service accounts is an upcoming enterprise feature, and is scheduled to become available in one of the next few releases.
+Service accounts is an enterprise feature available from 4.21 release.
```
Comment (nunogois) [inline website/docs/how-to/how-to-create-service-accounts.mdx:7]:
```suggestion
Service accounts is an enterprise feature available from Unleash 4.21 onwards.
```

File: src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap (hunk 3 above)
Comment (gastonfournier) [reply src/test/e2e/api/openapi/__snapshots__/openapi.e2e.test.ts.snap:8429]: yes!

**Review by nunogois — APPROVED** [review]
LGTM, small nitpicks we may want to have a look at.

**Pull request merged**